.github/workflows/ci.yml

```diff
@@ -142,6 +154,17 @@ jobs:
       PYTHONUTF8: 1
       TEST_RUNNER_TIMEOUT_FACTOR: 40
 
+    strategy:
+      matrix:
+        job-type: [standard, fuzz]
+        include:
+          - job-type: standard
+            generate-options: '-DBUILD_GUI=ON -DWITH_BDB=ON -DWITH_ZMQ=ON -DBUILD_BENCH=ON -DWERROR=ON'
+            job-name: 'Win64 native, VS 2022'
+          - job-type: fuzz
+            generate-options: '-DBUILD_FOR_FUZZING=ON -DWERROR=ON'
+            job-name: 'Win64 native fuzz, VS 2022'
```

Comment:
> It seems a bit annoying that the Windows build is now duplicated and seems to take 2h each times 2 = 4 CI hours total.

I assumed this is a caching issue? This shouldn't be doing anything different from the current windows CI...

Reply:
The majority of the time here is vcpkg compiling Qt modules from source, in the "Generate build system step": https://github.com/bitcoin/bitcoin/actions/runs/11705992609/job/32602049089?pr=31221. That should be getting cached in some way. Maybe push again and see if things are rebuilt?

Reply:
Re-pushed

Reply:
I would also think we can just turn off everything GUI related off for fuzzing?

Reply:
> Looks like it's rebuilding everything again, could it be that it can't store new caches on PRs?

The pkg cache is not stored on PRs (see [here](https://github.com/bitcoin/bitcoin/blob/2a0997b3a5dae13e1b42776123bc9a69781d2222/.github/workflows/ci.yml#L212)).

But I think the main problem was that I changed the job name from `win64-native` to `windows-native` which changed the lookup key for the cache (I assume).  Re-pushed again.

Reply:
"Cache restored successfully" 🎉. "Generate build system" only took 4min now. Resolving.